gef-ui/server/controllers/submit-to-ukef/index.js

```diff
@@ -64,7 +64,10 @@ const createSubmissionToUkef = async (req, res) => {
     await api.updateApplication(applicationId, application);
     await api.setApplicationStatus(applicationId, PROGRESS.SUBMITTED_TO_UKEF);
     // TODO: DTFS2-4706 - add a route and redirect instead of rendering?
-    return res.render('partials/submit-to-ukef-confirmation.njk', { applicationType: application.submissionType });
+    return res.render('partials/submit-to-ukef-confirmation.njk', {
+      applicationType: application.submissionType,
+      isNotice: isNotice(application.submissionType),
```

Comment:
need to update the unit test

Reply:
Apologies mate, should have checked. Have updated.